docs/wiki/EL/el-architecture.md

```diff
@@ -294,3 +321,6 @@ Read more about its [features](https://github.com/erthink/libmdbx#features). Add
 - [Engine Api Spec](https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#payload-validation) • [archived](https://web.archive.org/web/20250318111700/https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#payload-validation)
 - [Engine API: A Visual Guide](https://hackmd.io/@danielrachi/engine_api) • [archived](https://web.archive.org/web/20241006232802/https://hackmd.io/@danielrachi/engine_api)
 - [Engine API | Mikhail | Lecture 21](https://youtu.be/fR7LBXAMH7g)
+- ["Snapping Snap Sync: Practical Attacks on Go Ethereum Synchronising Nodes" (ETH Zurich)](https://appliedcrypto.ethz.ch/content/dam/ethz/special-interest/infk/inst-infsec/appliedcrypto/research/TavernaPaterson-SnappingSnapSync.pdf)
```

Comment:
also eth.org has these https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-layer-sync-modes

Reply:
nice source, added as well https://github.com/eth-protocol-fellows/protocol-studies/pull/437/commits/2bde312198c716340ca067a292936d77415c1ae6